docs/en/writing-running-appium/server-args.md

```diff
@@ -91,3 +91,4 @@ All flags are optional, but some are required in conjunction with certain others
 |`--keep-keychains`|false|[DEPRECATED] - (iOS-only) Whether to keep keychains (Library/Keychains) when reset app between sessions||
 |`--localizable-strings-dir`|en.lproj|[DEPRECATED] - (IOS-only) the relative path of the dir where Localizable.strings file resides |`--localizable-strings-dir en.lproj`|
 |`--show-ios-log`|false|[DEPRECATED] - (IOS-only) if set, the iOS system log will be written to the console||
+|`--enable-heapdump`|false|Enables NodeJS memory dumps collections feature. This feature is extremely useful for finding memory leaks. Use the 'kill -SIGUSR2 &lt;PID&gt;' command to create a dump of memory heap for _node_ process with the particular PID (this works for *nix systems only). The dump file will be created in the same folder, where main Appium script was executed and will have *.heapsnapshot extension. These snapshot can be then loaded to Chrome Inspector for further investigation. Read [this](https://blog.risingstack.com/finding-a-memory-leak-in-node-js/) article for more details.||
```

Comment:
I'd prefer the link to the Rising Stack article be "this article" (rather than "this").